### PR TITLE
added section on config file creation when installing python client

### PR DESCRIPTION
**Pull request by ychae**

### CODE REVIEW

File: articles/client_configuration.md [hunk 1]
```diff
@@ -56,6 +56,10 @@ The code block below displays an example config file with the available paramete
 
 <br/>
 
+## Create a .synapseConfig file on programmatic client installation
```
Comment (kdaily) [inline articles/client_configuration.md]: change title to something like 'Example Synapse client configuration'

File: articles/client_configuration.md [hunk 2]
```diff
@@ -56,6 +56,10 @@ The code block below displays an example config file with the available paramete
...
+
+When you install the [Synapse Python client](http://docs.synapse.org/python/#installation), a `.synapseConfig` file filled with examples, much like the sample above, should appear in your home directory if a config file is not already present. You can uncomment and fill out the parameters of interest to you to use.
```
Comment (kdaily) [inline articles/client_configuration.md]: What happens when you install `synapser`?

Comment (kdaily) [inline articles/client_configuration.md]: What happens when you install `synapser`?

Comment (kdaily) [reply articles/client_configuration.md]: change 'if a config file' to 'if a configuration file'

'of interest *for* you to use'

Comment (ychae) [reply articles/client_configuration.md]: Hmm not sure about the behavior of synapser but I'll try it out and edit the test to say yay or nay.

**Pull request merged**